src/madpack/madpack.py

```diff
@@ -674,7 +674,6 @@ def _process_py_sql_files_in_modules(modset, args_dict):
 
         # Loop through all SQL files for this module
         source_files = glob.glob(mask)
-        source_files = [s for s in source_files if '.setup' not in s]
```

Comment:
This line was included to ensure the common setup files for tests (like `madlib_keras_cifar.setup.sql_in`) are not treated as individual tests.

Reply:
Sorry for the late response,looks like this line was accidentally deleted,it has nothing to do with the fix

Reply:
No worries, I removed this line before merging the code. Thanks for the PR.

Reply:
Ok,thanks.